internal/rbd/rbd_util.go

```diff
@@ -1604,6 +1612,26 @@ func (ri *rbdImage) GetCreationTime(ctx context.Context) (*time.Time, error) {
 	return ri.CreatedAt, nil
 }
 
+// getLuks2HeaderSizeSet returns the value of the LUKS2 header size
+// set in the image metadata.
```

Comment:
From reading other parts of the code, it seems that the returned `headerSize` is in MiB? Would be good to mention that here.